_posts/2024-09-19-cheriot-dsbd-all-hands.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title:  "CHERIoT at the Digital Security by Design All Hands meeting"
+date:   2024-09-19
+categories: conference
+author: "David Chisnall"
+---
+
+Several companies presented CHERIoT-related things at the [Digital Security by Design](https://www.dsbd.tech) all-hands meeting yesterday!
+
+[lowRISC](https://lowrisc.org), whose [Sonata](https://cheriot.org/fpga/ibex/2024/06/10/sonata-quick-start.html) board was used by all of the demos, presented a demonstration of an automotive system where a bug in the volume control would overwrite the speed controller value (on a non-CHERI system).
```

Comment:
The source for the demo is at <https://github.com/lowRISC/sonata-software/tree/main/examples/automotive>.

There was also the snake demo which is fun and can be easily run on a single sonata board: <https://github.com/lowRISC/sonata-software/tree/main/examples/snake>. It's a little silly; it uses cheri bounds exceptions to catch when the snake leaves the screen.